theme/styles/snippets/email-subscribe.scss

```diff
@@ -60,6 +60,10 @@
     height: 100%;
     border: none;
 
+    &:hover {
```

Comment:
I think there should be a utility class for this built into basement, I think it's just `pointer`